applications/fake-services/app/templates/persistent-secret.yaml

```diff
@@ -0,0 +1,9 @@
+# apiVersion: v1
```

Comment:
should this just be deleted?

Reply:
argh that was my testing for other ticket, let me remove it.